bin/travis/before_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sudo install -o "$USER" -d /etc/blocktogether
+sudo install -o "$USER" -d /data/blocktogether
+cp -a "$(dirname ${BASH_SOURCE[0]})/../../"config/* /etc/blocktogether/
```

Comment:
I think I assumed `$0` had a different value, but it looks like `BASH_SOURCE` is slightly more reliable: http://stackoverflow.com/a/179231

But I'll add the `cd` to the top.  Should we also do `set -e`?